src/outputs/parthenon_hdf5.cpp

```diff
@@ -415,17 +488,26 @@ void PHDF5Output::WriteOutputFileImpl(Mesh *pm, ParameterInput *pin, SimTime *tm
     ~MPI_InfoDeleter() { MPI_Info_free(&info); }
   } delete_info{FILE_INFO_TEMPLATE};
 
-  PARTHENON_HDF5_CHECK(H5Pset_sieve_buf_size(acc_file, 262144));
```

Comment:
I disabled this by default, I can change that if needed. I need to look into the effects of always enabling this,  possibly with a poor choice of the value specified.